examples/dbus_client.py

```diff
@@ -0,0 +1,13 @@
+
+import dbus
+
+bus = dbus.SystemBus()
+bus.add_match_string("type='signal',sender='com.redhat.Blivet1',path_namespace='/com/redhat/Blivet1'")
```

Comment:
I don't quite understand what this is supposed to do. Maybe a comment would be nice here?